notebooks/remote_functions/remote_function_usecases.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>I imagine we should have some explanation about the condition and/or what happens with these references. For example, they won't change after the function is deployed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Added.</p>